geth/common/rpcall.go

```diff
@@ -0,0 +1,128 @@
+package common
+
+import (
+	gethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+)
+
+// RPCCall represents a unit of a rpc request which is to be executed.
+type RPCCall struct {
+	ID     int64
+	Method string
+	Params []interface{}
+}
+
+// ParseFromAddress returns the address associated with the RPCCall.
+func (r RPCCall) ParseFromAddress() gethcommon.Address {
+	params, ok := r.Params[0].(map[string]interface{})
+	if !ok {
+		return gethcommon.HexToAddress("0x")
```

Comment:
I see. I am not sure of implications if we change this... If you think it's good, let's leave it.